### ISSUE TITLE
draw polygon on picture

### ISSUE THREAD

**Issue by alikamrani** [yellowQ-software/yellowQ-Flutter-Image-Painter#28]
Hi, Is it possible to draw polygon on pic using this package. If it is not possible, is there any way to implement this?

**Comment by lively-bigyan** [comment]
Hi @alikamrani. Could you please specify it further? Do you mean to give `number of sides` as input to draw a polygon?

**Comment by alikamrani** [comment]
> Hi @alikamrani. Could you please specify it further? Do you mean to give `number of sides` as input to draw a polygon?

Exactly. Like line or rectangle tools, I want to add another tool so that a user can draw a polygon in the picture.

**Comment by lively-bigyan** [comment]
That is a pretty sweet idea and I have given a thought about the implementation. I could add it sometime in the coming days.
